packages/@lightningjs/ui-components/src/mixins/withSelections/withSelections.mdx

```diff
@@ -58,14 +62,6 @@ class Example extends lng.Component {
 }
 ```
 
-<Canvas>
-  <Story id="utilities-withselections--with-selections" />
-</Canvas>
-
-Take a look at the preview above. The row of modified [Buttons](?path=/docs/components-button--button) has navigation managed by the [Row](?path=/docs/components-row--row) component, which is also wrapped in `withSelections`. Navigate with the arrow keys to see the `focus` state change for each [Button](?path=/docs/components-button--button). Hit the `Enter` key to set the currently focused `Button` to be `isSelected`, giving it an underline. Also notice that underline of the previously selected `Button` is removed.
-
-In this example, `withSelections` is expecting the `Row` component to have a property called `items` which contains components that can handle an `isSelected` property. Fortunately, this describes the `Row` and `UnderlineButton` components perfectly! If your components are working with different property names, they can be customized with the `options` property. See the [Customization](#customization) section for options.
```

Comment:
Were these two paragraphs supposed to be deleted? Seems like a lot of context

Reply:
I originally removed it because it was referencing the story in the <Canvas>  code block above it. I had removed all the Canvas blocks in all the stories to clean up some of the documentation. I added the Canvas code block and this context. Maybe we can explore doing a deeper clean up of the documentation after the upgrade is complete.